File: test-service-java/src/main/java/com/example/testservicejava/v1/entities/Geo.java
package com.example.testservicejava.v1.entities;

import com.fasterxml.jackson.annotation.JsonProperty;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "geos")
@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class Geo {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    @JsonProperty(value = "id")
    private Long id;

    @Column(name = "lat")
    @JsonProperty(value = "lat")
    private float lat;

    @Column(name = "lng")
    @JsonProperty(value = "lng")
    private float lng;

}

File: test-service-kotlin/src/main/kotlin/org/xapps/service/testservice/v1/entities/Geo.kt
package org.xapps.service.testservice.v1.entities

import com.fasterxml.jackson.annotation.JsonProperty
import jakarta.persistence.*

@Entity
@Table(name = "geos")
open class Geo(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    @JsonProperty(value = "id")
    var id: Long = 0,

    @Column(name = "lat")
    @JsonProperty(value = "lat")
    var lat: Float = 0.0f,

    @Column(name = "lng")
    @JsonProperty(value = "lng")
    var lng: Float = 0.0f
)